### PR TITLE
socat-forwarder image

### PR DESCRIPTION
**Pull request by chemikadze**
Image that provides simple TCP forwarding facilities. Command-line option format is the same as for SSL -L flag:

```
docker run -d -P --expose 80 chemikadze/socat-forwarder -L 80:ya.ru:80
```

### CODE REVIEW

**Comment by rtimush** [comment]
Do we really need haproxy? From the first sight it is very similar to the [ambassador pattern](https://docs.docker.com/articles/ambassador_pattern_linking/) so may be a lightweight `socat` is enough?

**Comment by chemikadze** [comment]
Oh, socat is good alternative, thanks.

**Comment by chemikadze** [comment]
@rtimush Reimplemented using socat.

File: socat-forwarder/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:latest
```
Comment (rtimush) [inline socat-forwarder/Dockerfile:1]: Probably basing on busybox would be better

**Comment by rtimush** [comment]
Looks good to me

File: socat-forwarder/Dockerfile (hunk 1 above)
Comment (chemikadze) [inline socat-forwarder/Dockerfile:1]: I tried to, but busybox xargs cannot into -P.

**Pull request merged**